processing/src/main/java/org/apache/druid/segment/QueryableIndexStorageAdapter.java

```diff
@@ -270,7 +270,8 @@ public VectorCursor makeVectorCursor(
       VirtualColumns virtualColumns,
       Granularity gran,
       boolean descending,
-      @Nullable QueryMetrics<?> queryMetrics
+      @Nullable QueryMetrics<?> queryMetrics,
+      boolean useInMemoryBitmapInQuery
```

Comment:
`useInMemoryBitmapInQuery` is not used inside this function. Is this the final version ? This new parameter introduces lots of changes in other files.

Reply:
Hi Frank, this case is caused by `IncrementalIndexStorageAdapter` and `QueryableIndexStorageAdapter` extending the same interface and lots of other classes implementing the same interface introduced changes accordingly as well. However, based on the previous discussion on the necessity of `useInMemoryBitmapInQuery`, this param probably can be omitted and just use `enableInMemoryBitmap` as the only switch, let me remove this param and simplify the code.

Reply:
Understood.

Reply:
Removed `useInMemoryBitmapInQuery` from the code which removed about ~30 changed files.